lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -60,19 +61,35 @@ $-separator-color: var(--nx-swatch-indigo-40);
   color: $-text-color;
   flex: none;
   margin: 0;
+  position: relative;
 
   &:focus:not(:active) {
-    outline-color: $-focus-color;
-    padding-top: calc(var(--nx-spacing-base) - #{$-button-border-width});
-    padding-bottom: calc(var(--nx-spacing-base) - #{$-button-border-width});
+    outline-color: var(--nx-color-global-sidebar-menu-icon-border-focus);
+    padding-top: calc(var(--nx-spacing-base) - var(--nx-border-width));
+    padding-bottom: calc(var(--nx-spacing-base) - var(--nx-border-width));
```

Comment:
Are these lines necessary anymore? It seems like they match the padding from the `small-icon-btn` mixin which they override.

Reply:
> Are these lines necessary anymore? It seems like they match the padding from the `small-icon-btn` mixin which they override.

Nope. Removed: 3d04e002f544f5deb0f4652f5170a9cc5f27a6e5